vue2-vuetify/src/complex/OneOfRenderer.vue

```diff
@@ -42,18 +42,28 @@
 
     <v-dialog v-model="dialog" persistent max-width="600" @keydown.esc="cancel">
       <v-card>
-        <v-card-title class="text-h5"> Clear form? </v-card-title>
+        <v-card-title class="text-h5">
+          {{ t('form.clear.title', 'Clear form?') }}
```

Comment:
This should ideally then work in a similar way like the mentioned PR above.

Reply:
I couldn't find the properties for that in the jsonforms core yet or maybe I'm not looking at the right place - can you point me to the locations in the jsonforms core where this is defined the same way how we have the arrayTranslations ?

Reply:
The idea would be to handle this via the binding and pass down the translated labels instead of calling the translator within the renderer. But for now I'm fine with this.